<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameDemo</class>
 <widget class="QFrame" name="FrameDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>2</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>1200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Demo</string>
     </property>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>sent_white_screen:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>输出正常图像到Tcon</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup1</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>输出纯白色图像到Tcon</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup1</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>输出BMA图像到Tcon</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup1</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>dcon_adjust_bypass:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>正常画面</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup2</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>打全屏全白画面</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup2</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>打左半屏全白画面</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup2</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>打右半屏全白画面</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup2</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>打固定窗口全白画面</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup2</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>select_area_enable:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxSelectArea">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>30</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Screen width:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>105</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Screen height:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSrcW">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>70</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>600</string>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSrcH">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>105</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>480</string>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>150</y>
        <width>720</width>
        <height>405</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 83, 83);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSetEara">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>100</y>
        <width>81</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Y1">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>105</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>70</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>X1:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_X1">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>70</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>105</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Y1:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_X2">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>70</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>600</string>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>70</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>X2:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>105</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Y2:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Y2">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>105</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>480</string>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxFullSrceen">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>65</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>选择全屏</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup1"/>
  <buttongroup name="buttonGroup2"/>
 </buttongroups>
</ui>
